examples/wallet/app/containers/App.js

```diff
@@ -10,6 +10,6 @@ export default class App extends React.Component<Props> {
 
   render() {
     const { children } = this.props;
-    return <React.Fragment>{children}</React.Fragment>;
+    return <>{children}</>;
```

Comment:
Why do you return an empty tag?

Reply:
It's the same as before, but with a [shorthand syntax](https://reactjs.org/docs/fragments.html#short-syntax) which the new version of eslint prefers.